contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-api.mdx

```diff
@@ -102,6 +102,33 @@ When called, `decide` returns the flag values and payloads, along with more info
 > 
 > This enables partial updates to currently active flags in your clients.
 
+#### Quota Limiting
+
+If your organization exceeds its feature flag quota, the `/decide` endpoint will return a modified response:
```

Comment:
Just clarifying when we are looking at here as most people won't be familar with what the decide response looks like :)
```suggestion
If your organization exceeds its feature flag quota, the `/decide` endpoint will return a modified response with `quotaLimited`:
```